README.md

```diff
@@ -129,27 +129,22 @@ Ask for help on the IRC channel: #cordova on irc.freenode.net.
 Writing a Blog Post
 ===================
 
-Use the grunt scripts!
-
-    # Clone apache-blog-posts repo if you haven't already
-    $ grunt cloneBlog
-
-    # Create your blog posts in the apache-blog-posts (or on github) directory, and share with other for review via `git commit`. If creating a new md file, use an earlier one as a template.
-
-    # pull down the latest shared edits of apache-blog-posts
+    # Pull down the latest website codebase for the current posts
     git pull
 
-    # Edit your md file to remove undesired markdown links. If there is a phrase in square brackets that isn't a CB-xxxx reference, escape it with backslashes. Otherwise, heruko might error out and fail to build all the html.
+    # Create a new entry in the www/_posts directory.
+
+    # Use an earlier post an a template. Edit your md file to remove undesired markdown links. If there is a phrase in square brackets that isn't a CB-xxxx reference, escape it with backslashes. Otherwise, heruko might error out and fail to build all the html.
     [CB-1234] \[iOS\] \[Camera\] add a whizzbang to the snarfblat
 
     # Set a marker where the summary on the home page should stop displaying. Add the following html comment line to your md file at the desired cutoff point:
     <!--more-->
 
-    # in the front matter of your blog entry, set the `date:` field to the desired date that you want to appear near the title. Be aware that the date (explicit here or implied via the filename) will be used to generate the relative path to this html file (i.e., "/announcements/2014/09/22/cordova-361.html"), as will the `categories:` front matter value.
+    # In the front matter of your blog entry, set the `date:` field to the desired date that you want to appear near the title. Be aware that the date (explicit here or implied via the filename) will be used to generate the relative path to this html file (i.e., "/announcements/2014/09/22/cordova-361.html"), as will the `categories:` front matter value.
     date: 2014-09-22
     categories: releases
 
-    # git commit your changes to apache-blog-posts
+    # Preview it locally by running the site using gulp
 
```

Comment:
Please make this whole block not be a code block, except for the pieces of code. It was just poorly formatted before.